src/stdlib.jl

```diff
@@ -1,25 +1,25 @@
 import Base.CoreLogging:
     AbstractLogger,
+    LogLevel,
     handle_message,
     min_enabled_level,
     shouldlog,
-    global_logger,
-    Debug
+    global_logger
 
-const LEVEL_MAP = Dict{AbstractString}
```

Comment:
Curious if this was just never used

Reply:
I think it was used in my initial prototype code, but not since this has been released in Memento.